02-intermediate-ruby/07-enumerable.md

```diff
@@ -113,9 +113,13 @@ Use these methods to filter collections by a conditional evaluation. `select` wi
 ```
 
 ## There's so much more to discover
-`Enumerable` is one of my favorite things about Ruby. It is immensely useful. With it, we can write expressive, highly semantic code that is also concise. Spend some time with the [Enumerable docs](http://ruby-doc.org/core-2.3.0/Enumerable.html). Good knowledge in here will serve you in every application you build, And not just with Ruby. The methods contained in `Enumerable` describe ideas and patterns you'll find in every programming discipline (and technical interview, just sayin').
+`Enumerable` is one of the most exciting things about Ruby. It is immensely useful. With it, we can write expressive, highly semantic code that is also concise. Spend some time with the [Enumerable docs](http://ruby-doc.org/core-2.4.0/Enumerable.html). Good knowledge in here will serve you in every application you build, and not just with Ruby. The methods contained in `Enumerable` describe ideas and patterns you'll find in every programming discipline.
+
+## Activity: Refactor Loops
+Let's take what we just learned about `Enumerable` and apply that to the [Loop Problem Practice](https://github.com/AdaGold/standalone-exercises/blob/master/loop-problem-practice.md) that we did previously. Work with your seat squad to refactor each problem to use one of these `Enumerable` methods we just learned about.
 
 ## Activity: Divide & Conquer Enumerable
+In a group of 4, you shall research each of these `Enumerable` methods. Prepare a short (<5 minute) demonstration to present to the group.
```

Comment:
Are each group researching all the methods or is each group researching one method and presenting that method?